### DISCUSSION TITLE
Include Postgres command helpers

### DISCUSSION THREAD

**Discussion by morrislaptop** [beyondcode/herd-community#742] [Ideas]
The command utilities like `createdb` and `pg_restore` are very handy. 

In fact, if you've used Laravel Schema Dump, they will no longer work as it relies on these commands to exist..

**Comment by morrislaptop** [comment]
The solution is to `brew install libpq` and `brew link libpq --force` but I was really looking forward to not using Homebrew anymore!

**Reply by mpociot** [reply]
The service binaries are located at /users/Shared/Herd/services/...
You might need to manually add a folder to your path. 
We already symlink a couple of binaries when installing PostgreSQL but not all of them.

**Comment by morrislaptop** [comment]
Thanks!
